extensions/parcel/package.json

```
@@ -0,0 +1,300 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "parcel",
+  "title": "Parcel",
+  "description": "Allow Parcel users to see upcoming and active deliveries",
+  "icon": "extension-icon.png",
+  "author": "marcmagn1",
+  "categories": [
+    "Productivity",
+    "Applications"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "view-my-deliveries",
+      "title": "View My Deliveries",
```

```suggestion
      "name": "my-deliveries",
      "title": "My Deliveries",
```

This naming is more consistent with commands of popular extension maintained by Raycast team (e.g., "My Reminders", "My Schedule", "My Pull Requests").